src/main/java/org/gridsuite/network/map/dto/definition/tieline/TieLineTabInfos.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.dto.definition.tieline;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.powsybl.iidm.network.Country;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.network.map.dto.ElementTabInfoWithProperties;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@SuperBuilder
+@Getter
+public class TieLineTabInfos extends ElementTabInfoWithProperties {
+
+    private String voltageLevelId1;
```

Comment:
Some attributes don't have **@JsonInclude(JsonInclude.Include.NON_NULL)** .Is there a reason for that?
For example : voltageLevelId1 and voltageLevelId2

Reply:
The attributes that don't have the Json include non null annotation are the attributes that can't be null ...